indexers/keyvalue/LevelDBIndexer/__init__.py

```diff
@@ -46,14 +47,37 @@ def get_query_handler(self):
         import plyvel
         return plyvel.DB(self.index_abspath, create_if_missing=True)
 
-    def query(self, key: str, *args, **kwargs) -> Optional['jina_pb2.Document']:
+    def query(self, key: Any) -> Optional[Any]:
         """Find the protobuf chunk/doc using id
 
         :param key: ``id``
         :return: protobuf chunk or protobuf document
         """
-        v = self.query_handler.get(key.encode('utf8'))
+        v = self.query_handler.get(bytes(key))
         value = None
         if v is not None:
-            value = Parse(json.loads(v.decode('utf8')), jina_pb2.Document())
+            value = Parse(json.loads(v.decode('utf8')), Document())
         return value
+
+    # TODO remove this method once https://github.com/jina-ai/jina/pull/1380 is merged
```

Comment:
Have you tested whether it passes with that method? Does this class have the same internal behavior?

Reply:
I couldn't use this base class method at the end, since it requires the `self.query_handler.header` attribute which does not exist in that indexer.
So I removed the reference to the ticket.